.github/workflows/automerge.yml

```diff
@@ -0,0 +1,34 @@
+name: automerge
+on:
+  pull_request:
+    types:
+      - labeled
+      - unlabeled
+      - synchronize
+      - opened
+      - edited
+      - ready_for_review
+      - reopened
+      - unlocked
+  pull_request_review:
+    types:
+      - submitted
+  check_suite:
+    types:
+      - completed
+  status: {}
+jobs:
+  automerge:
+    runs-on: ubuntu-latest
+    steps:
+      - name: automerge
+        uses: "pascalgn/automerge-action@135f0bdb927d9807b5446f7ca9ecc2c51de03c4a"
+        env:
+          GITHUB_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
+          # https://github.com/pascalgn/automerge-action#configuration
+          MERGE_LABELS: "automerge,!wip"
+          MERGE_METHOD: "squash"
+          MERGE_REMOVE_LABELS: "automerge,wip"
+          MERGE_FORKS: true
+          UPDATE_LABELS: "" # All branches are automatically kept up-to-date
```

Comment:
Can we add labels to enable/disable auto-updating as well? I have my remotes setup so that I can manually merge from origin/master, but its extra hoops if Github has modified my fork. And I think we should disable it for `wip` as well.

Reply:
Good point. I've added a new `sync` label for this (and `wip` prevents updating). I think it's better to keep the `automerge` and auto-update (`sync`) flags different for now.